styles.css

```diff
@@ -1,62 +1,66 @@
 body {
-  margin: 0;
-}
-
-.icon {
   display: flex;
-  justify-content: flex-end;
-  align-items: flex-end;
-  margin-right: 10%;
-}
-
-.section1 {
-  background-image: url('img/header-illsutration-mobile.png');
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  background-image: url('img/Header-llustration-desktop.png');
   background-repeat: no-repeat;
-  background-size: contain;
-  background-position: center top -15px;
-  height: 100vh;
-  width: 100%;
-  padding-top: 50px;
+  background-size: cover;
+  background-size: 100% auto;
```

Comment:
- [ ] Great work implementing the required changes from the previous reviewer. However, there're few things you need to work on.
- [ ] Your desktop version does not look like figma template. Kindly give a closer look to the figma file and walk your design alongside the figma template.
- [ ] The css background image you used on some sections of your page were not properly positioned and as result distorted the whole design on the page. Kindly look into this and fix.

### Your Design
[your-app.webm](https://github.com/NgachaIan/Personal_Portfolio/assets/26597432/25c38597-f2c4-4ecc-96f7-f2cfe29a5f85)

### Figma Design
[fig-app.webm](https://github.com/NgachaIan/Personal_Portfolio/assets/26597432/5d912f19-a9a8-42bd-b117-e3a8900743e9)